Gemfile

```diff
@@ -78,10 +79,16 @@ group :development, :test do
   gem 'capybara'
 end
 
-gem 'sentry-ruby'
-gem 'sentry-rails'
+gem "sentry-ruby"
+gem "sentry-rails"
 gem 'grape', '~> 1.7'
 gem 'grape-entity'
 gem 'grape_on_rails_routes'
 gem 'rack-cors'
 gem 'uuidtools', '~> 2.1', '>= 2.1.5'
+
+# For Windows only
+gem 'sys-proctable' if RUBY_PLATFORM =~ /mingw32/
+gem 'fast_stack' if RUBY_PLATFORM =~ /mingw32/
+gem 'wdm', '>= 0.1.0' if Gem.win_platform?
+gem 'tzinfo-data' if RUBY_PLATFORM =~ /mingw32/
```

Comment:
There are still these Windows Only Gems in the Gemfile.

Reply:
It is there to make Phoenix compatible with Windows, but also maintaining compatibility with Linux at the same time.

Reply:
@madoleary thoughts?

Reply:
Hi guys, I apologize for the delay, I've been held up with other tasks. I've done some research on how Bundler treats these types of situations and it seems there's a more canonical way to do it : https://bundler.io/v2.2/man/gemfile.5.html#PLATFORMS

@EarldridgeJazzedPineda , could you try it Bundler's way?

Reply:
Yeah sure